### PR TITLE
Styled components and animation

### PR DESCRIPTION
**Pull request by swalk001**

### CODE REVIEW

**Review by stanley30543 — APPROVED** [review]
I think it looks fine

File: client/index.jsx [hunk 1]
```diff
@@ -9,20 +9,67 @@ const Body = styled.div`
...
+  left: -10%;
+  top: 40%;
+  opacity: .2;
+  transform: translateY(-50%);
+  transition: opacity .2s,right .2s,left .2s;
+  :hover {
+    cursor: pointer;
+    opacity: .9;
+    background-color: white;
+  }
```
Comment (ryancmoniz) [inline client/index.jsx]: These 2 buttons share a lot of similar properties that you end up repeating multiple times. Have you thought about creating a base button component and then overriding or adding only the things that are different between the base button and  the extension?

File: client/index.jsx [hunk 2]
```diff
@@ -130,19 +177,33 @@ class App extends React.Component {
...
+              }
+              return (
+                <Thumbnails
+                  onClick={this.thumbClick}
+                  key={image._id}
+                  src={image.url}
+                />
+              );
+            })
+        }
```
Comment (ryancmoniz) [inline client/index.jsx]: Could we consider doing some of this work outside of our return and getting some better readability out of it?

File: server/index.js [hunk 3]
```diff
@@ -7,7 +7,7 @@ const db = require('../db/index.js');
 app.use(express.static(path.join(__dirname, '/../public/dist')));
 
 app.get('/item', (req, res) => {
-  const itemId = req.query.id || 20;
+  const itemId = req.query.id || Math.floor(Math.random() * 50) || 36;
   db.retrieveListingImages(itemId, (results) => {
     res.json(results);
   });
```
Comment (ryancmoniz) [inline server/index.js:13]: If there was an error with the database retrieving images, what would happen here?

**Review by ryancmoniz — COMMENTED** [review]
Lets remember to not commit our bundle.js file. It is also a good idea to remove your console.logs once something is working correctly. Left a few other comments, otherwise LGTM!

**Pull request merged**